### DISCUSSION TITLE
Boost_INCLUDE_DIR not found

### DISCUSSION THREAD

**Discussion by igorbrai** [osm2pgsql-dev/osm2pgsql#1457] [Q&A]
While trying to install Nominatim 3.6.0 i get an error that Boost_INCLUDE_DIR is set to not found and it can not find it while making osm2pgsql, any ideas?
```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
Boost_INCLUDE_DIR
   used as include directory in directory /home/name/nominatim/osm2pgsql
   ```

**Comment by lonvia** [comment]
Please read and follow https://nominatim.org/release-docs/3.6.0/appendix/Install-on-Ubuntu-20/#installing-the-required-software Make sure you install **all** mentioned dependencies.
